src/oiiotool/oiiotool.cpp

```diff
@@ -4940,7 +4952,7 @@ OIIOTOOL_OP(contrast, 1, [&](OiiotoolOp& op, span<ImageBuf*> img) {
 
 // --box
 // clang-format off
-OIIOTOOL_OP(box, 1, nullptr, ([&](OiiotoolOp& op, span<ImageBuf*> img) {
+OIIOTOOL_INPLACE_OP(box, 1, nullptr, ([&](OiiotoolOp& op, span<ImageBuf*> img) {
     img[0]->copy(*img[1]);
```

Comment:
Just checking: since the Op now runs in-place, do we still need this copy ?

Reply:
Internal to copy, it checks if it's doing a self-assignment and takes an early out. So there is not actually a needless copy here. 

But that's a good point, maybe for clarity it's better to remove it. Let me get rid of it and run a test to be sure we're correct that it can go.

Reply:
Yes, you were right, I'm able to eliminate that line, as it is not currently necessary. Will push an update.